data-prepper-core/src/test/java/org/opensearch/dataprepper/acknowledgements/DefaultAcknowledgementSetManagerTests.java

```diff
@@ -38,17 +39,24 @@ class DefaultAcknowledgementSetManagerTests {
     @Mock
     JacksonEvent event3;
 
-    DefaultEventHandle eventHandle1;
-    DefaultEventHandle eventHandle2;
-    DefaultEventHandle eventHandle3;
-    Boolean result;
+    private PluginMetrics pluginMetrics;
+    private DefaultEventHandle eventHandle1;
+    private DefaultEventHandle eventHandle2;
+    private DefaultEventHandle eventHandle3;
+    private DefaultEventHandle eventHandle4;
+    private DefaultEventHandle eventHandle5;
+    private DefaultEventHandle eventHandle6;
```

Comment:
Nit: Could make this a `List<DefaultEventHandle>` just to make it more flexible for future test case additions